apis/v1beta1/common.go

```diff
@@ -115,10 +115,6 @@ type OpenTelemetryCommonFields struct {
 	// Replicas is the number of pod instances for the underlying replicaset. Set this if you are not using autoscaling.
 	// +optional
 	Replicas *int32 `json:"replicas,omitempty"`
-	// Autoscaler specifies the pod autoscaling configuration to use
-	// for the workload.
-	// +optional
-	Autoscaler *AutoscalerSpec `json:"autoscaler,omitempty"`
```

Comment:
Theoretically someone may want an auto scaler for the TA, no?

Reply:
Theoretically, yes, but I don't really want to support it just based on that possibility. Right now scaling the TA only really scales the HTTP server, and that's really not a very performance-sensitive component. So I'd like to avoid it until someone actually asks.

Reply:
Theoretically, yes, but I don't really want to support it just based on that possibility. Right now scaling the TA only really scales the HTTP server, and that's really not a very performance-sensitive component. So I'd like to avoid it until someone actually asks.